flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -80,7 +80,6 @@
 import java.util.Date;
```

Comment:
Can you rephrase my `mid work` commit message to:

> [FLINK-12143][dist] Ship plugins in the cluster the same way as lib jars

Reply:
I have merged some commits together + renaming.